### ISSUE TITLE
Compatibility with python 3.8.10 broken

### ISSUE THREAD

**Issue by brunob** [Taiko2k/TauonMusicBox#972]
Hi @Taiko2k 

This commit https://github.com/Taiko2k/TauonMusicBox/commit/6db8bd019be777530398d8097ba6d9e1527b9e32#diff-985f03dd5066cc942ac56fadc1697989d89d40308fc2b1f37a4a990579cdd916 produce an error with python < 3.10 since the match/case syntax was introduced in python 3.10 ref https://stackoverflow.com/a/69726064

**Comment by Taiko2k** [comment]
Yes I noted that Python 3.10 is now required on the release notes page. https://github.com/Taiko2k/TauonMusicBox/releases/tag/v7.6.3

**Comment by brunob** [comment]
Damn, sorry for the noise, i have now to find a way to get a most recent python on my old ubuntu 20.04 :\

**Comment by brunob** [comment]
Not so easy, after hours i can't get gi module up to date and compatible with python 3.11. Think i'll stick on 7.6.2 tag before upgrading my system...
